src/slot_ranges.c

```diff
@@ -25,7 +25,7 @@ struct SharedSlotRangeArray {
 // Dropped when Slots_DropCachedLocalSlots is called (when we know local slots have changed)
 static SharedSlotRangeArray *localSlots = NULL;
 
-const SharedSlotRangeArray *Slots_GetLocalSlots(void) {
+SharedSlotRangeArray *Slots_GetLocalSlotsMutable() {
```

Comment:
Unexpected, but this PR is temporary, so I'll approve this.
But for the record, this is a shared object, so it should never be mutable (apart from internal reference counting and freeing)